x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/transform_search_bar_filters.tsx

```diff
@@ -45,6 +47,25 @@ export const transformFilters: SearchFilterConfig[] = [
       ),
     })),
   },
+  {
+    type: 'field_value_selection',
+    field: 'health',
+    name: i18n.translate('xpack.transform.healthFilter', { defaultMessage: 'Health' }),
+    multiSelect: false,
+    options: Object.values(TRANSFORM_HEALTH).map((val) => ({
+      value: val,
+      name: val,
+      view: (
+        <EuiBadge
```

Comment:
Did you consider using `EuiHealth` here to add some text next to the badge to provide additional context to the color of the `EuiBadge` - see [guidelines](https://elastic.github.io/eui/#/display/health). For the text could use `Healthy` `Warning` `Failure` (maybe) and `Unknown` ?

Reply:
What I meant to say here was that it might be helpful to associate 'status' strings with these badges rather than the names of the colors, such as `Healthy` `Warning` `Failure` and `Unknown`.

Reply:
Updated in 750faf8, also updated the screenshots in the PR description.